appveyor.yml

```diff
@@ -0,0 +1,42 @@
+version: 1.0.{build}
+max_jobs: 2
+image: Visual Studio 2015
+build_script:
+- cmd: >-
+    git submodule init
+
+    git submodule update
+
+    nuget restore ReactWindows\ReactNative.sln
+
+    set DEVENV="%VS140COMNTOOLS%\..\IDE\devenv"
+
+    %DEVENV% /build "Debug|x64" ReactWindows\ReactNative.sln
+
+    npm install -g npm
+
+    npm version
+
+    npm install -g react-native-cli
+
+    npm install
+
+    mkdir ReactWindows\Playground\ReactAssets
+
+    react-native bundle --platform windows --entry-file ReactWindows/Playground/index.windows.js --bundle-output ReactWindows/Playground/ReactAssets/index.windows.bundle --assets-dest ReactWindows/Playground/ReactAssets/
```

Comment:
Why do we need to test this?  The bundle command comes exclusively from facebook/react-native